edb/schema/ordering.py

```diff
@@ -776,8 +776,18 @@ def _extract_op(stack: Sequence[sd.Command]) -> List[sd.Command]:
         parent_op = alter_delta
         new_stack.append(parent_op)
 
-    stack[-2].discard(stack[-1])
-    parent_op.add(stack[-1])
-    new_stack.append(stack[-1])
+    op = stack[-1]
+    # We don't want duplicate nested Alters on the same object (which
```

Comment:
Hm, how hard would it be to prevent this artifact in the first place?  ISTM we might be throwing away data by pruning it like this.

Reply:
So the situation where it comes up here was caused by `150b143d09dab20c31876a7636d31b091ba7ee5c`, which moved the AlterProperty/AlterOwned for a property underneath its delete, which results in a doublly nested AlterProperty once `_extract_op` messes around with things.

I think we can just directly attach the AlterOwned to the Delete, though, without an intermediate Alter, which will neatly solve the problem.